backend/clubs/models.py

```diff
@@ -1895,19 +1904,20 @@ def send_confirmation_email(self):
         Send a confirmation email to the ticket owner after purchase
         """
         owner = self.owner
+        qr_url = f"https://{settings.DOMAINS[0]}/api/tickets/{self.id}/qr"
 
         output = BytesIO()
         qr_image = self.get_qr()
         qr_image.save(output, format="PNG")
-        decoded_image = base64.b64encode(output.getvalue()).decode("ascii")
+        image = base64.b64encode(output.getvalue())
```

Comment:
Is there a reason we're still base64 encoding this...? Wouldn't `msg.attach` abstracts that away in a sane universe?